test/integration/component/test_multiple_nic_support.py

```diff
@@ -224,7 +224,7 @@ def setUpClass(cls):
                     cls.virtual_machine1.default_network_id = nic.networkid
                     break
         except Exception as e:
-            cls.fail("Exception while deploying virtual machine: %s" % e)
+            cls.fail("Exception while deploying virtual machine: %s" % (e))
```

Comment:
```suggestion
            cls.fail(f"Exception while deploying virtual machine: {e}")
```
not necessary but nice ;)

Reply:
@DaanHoogland there's an `f"Ex...` is that a typo or something new in py3?

Reply:
Nevermind, I googled - I need a bit a re-learning py3 https://realpython.com/python-f-strings/